pykern/pkcli/test.py

```diff
@@ -188,8 +194,14 @@ def _try(output, restartable):
                     env=_env(restartable),
                 )
                 o = pkio.read_text(output)
-                if _COROUTINE_NEVER_AWAITED.search(o):
-                    pkio.write_text(output, o + _FAILED_ON_WARNINGS)
+                if m := _COROUTINE_NEVER_AWAITED.search(o):
+                    pkio.write_text(
```

Comment:
I don't think you need to write the text. Just append to output.

Reply:
Don't I need to write to output since it is the log filename and if we find the runtimewarning in the log we have to write to that log file? Am I getting something mixed up?

Reply:
I see. You are rewriting the output to the log. I think we should just append to the log, not rewrite it. You could just add that failure and not remove the passed part of it.